all.json

```diff
@@ -1,6 +1,25 @@
 {
 	"allow": [],
 	"deny": [
+		"test1.consultando.eu",
```

Comment:
Seems to be a WP login?

Reply:
Yeah, but the full path redirects to malicious content. Gonna remove it.